content/applications/sales/subscriptions/reports.rst

```diff
@@ -1,120 +1,397 @@
-=======
-Reports
-=======
+====================
+Subscription reports
+====================
 
-As we know, understanding how our business is going, and where it is going, is key to success. And
-particularly so when we offer subscription services or products.
+Understanding how well subscriptions are performing is of the utmost importance.
 
-Before getting to the heart of the matter, it is very important to recall certain essential
-concepts to the proper understanding of the following reports:
+The Odoo *Subscriptions* app provides a collection of detailed, fully-customizable reports
+showcasing various subscription metrics and analytics. This knowledge can be used to improve
+decision-making and enhance subscription-related strategies.
 
-- **Monthly Recurring Revenue (MRR)**: MRR is arguably the most important metric for subscription
-  businesses. It shows the monthly revenue earned with subscription-based products or services. It
-  is a consistent number used to track all recurring revenue over time, in monthly increments.
+Reporting page elements
+=======================
 
-- **Annual Run-Rate (ARR)**: ARR is the yearly version of MRR, which is based on the current MRR,
-  to estimate the coming year's performance. However, this estimation does not take variations and
-  growth into account.
+All subscription-based reporting pages can be accessed via the :guilabel:`Reporting` header
+menu in the *Subscriptions* application.
 
-.. image:: reports/difference-between-MRR-and-ARR.png
+The following describes elements found on each reporting page.
+
+Filters and Group By
+--------------------
+
+Filters are used to narrow down metrics to show specific analytics, whereas groupings (via the
+:guilabel:`Group By` option) are used to gather specific sections' data into groups for more
+organized analysis.
+
+This section refers to both filters and groupings, as a combination of the two can be saved in the
+:guilabel:`Favorites` section.
+
+To modify the results being shown on any reporting page, click the :guilabel:`▼ (downward facing
+arrow)` icon to the right of the search bar. Doing so reveals a drop-down menu of detailed filter
+and grouping options.
+
+.. note::
+   Multiple filter options and grouping options can be selected at once.
+
+.. image:: reports/reporting-filter-options-dropdown.png
   :align: center
```

Comment:
Indent image attributes by one more space, please check all of the images in this doc, there are quite a few that need this adjustment :)
```suggestion
.. image:: reports/reporting-filter-options-dropdown.png
   :align: center
   :alt: The reporting filter drop-down menu of options in the Odoo Subscriptions application.
```